.github/workflows/ci.yml

```diff
@@ -14,7 +14,6 @@ jobs:
           cache: 'npm'
 
       - run: npm ci
-      - run: npm run build
```

Comment:
Why don't we need to run the `build` in these workflows anymore?

Reply:
since we're using `tsup` and not `tsc` to build and now that we're running `tsc` in our lint step, the `build` step isn't required here!